lib/v2/5eplay/index.js

```diff
@@ -0,0 +1,83 @@
+const cheerio = require('cheerio');
+const zlib = require('zlib');
+const got = require('@/utils/got');
+const { parseDate } = require('@/utils/parse-date');
+const { getAcwScV2ByArg1 } = require('./utils');
+
+module.exports = async (ctx) => {
+    const rootUrl = 'https://csgo.5eplay.com/';
+    const apiUrl = `${rootUrl}api/article?page=1&type_id=0&time=0&order_by=0`;
+    const articleUrl = `${rootUrl}article`;
+
+    const { data: response } = await got({
+        method: 'get',
+        url: apiUrl,
+    });
+
+    const expiredTime = 25 * 60 * 1000;
+
+    let acw_sc__v2 = '';
+    // get arg1
+    await ctx.cache.tryGet(
```

Comment:
This `tryGet` function has no return value which means nothing will be stored in the cache.
Beside that, `acw_sc__v2` will be `''` when cache-hit.
docs: https://docs.rsshub.app/joinus/use-cache.html